py/vtdb/tablet.py

```diff
@@ -90,7 +90,7 @@ def __str__(self):
     return '<TabletConnection %s %s %s/%s>' % (
         self.addr, self.tablet_type, self.keyspace, self.shard)
 
-  def dial(self):
```

Comment:
Is it necessary to hardcode a default? And even so, we shouldn't use youtube-specific defaults.